go.mod

```diff
@@ -44,12 +43,12 @@ require (
 	github.com/spf13/viper v1.3.2
 	github.com/status-im/keycard-go v0.0.0-20191119114148-6dd40a46baa0 // indirect
 	github.com/tyler-smith/go-bip39 v1.0.2 // indirect
-	golang.org/x/net v0.0.0-20200301022130-244492dfa37a
+	golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58
 	google.golang.org/appengine v1.6.5 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 )
 
-replace github.com/ethereum/go-ethereum => github.com/makerdao/go-ethereum v1.9.11-rc2
+replace github.com/ethereum/go-ethereum => github.com/makerdao/go-ethereum v1.5.10-0.20200521202712-d5657e3ff589
```

Comment:
This seems like a low version? If we own it, maybe we want to bump the name and/or make sure it's rebased on a more recent version?

Reply:
Good call, I'm really not sure why the version is so low. 🤔  I'll try cutting a release of the new new geth patch and use that instead.